planetarium/NineChronicles
# Fix bug in combination tab transition.

### Description

SSIA

### How to test

1. Switch category tabs randomly in combination window and close it.
2. Join combination window again and check if initial category tab is switched properly.

### Required Reviewers
@planetarium/9c-dev @Namyujeong
